debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -267,21 +270,44 @@ public Event nextEvent(ByteArrayInputStream inputStream) throws IOException {
         // Add our custom deserializers ...
         eventDeserializer.setEventDataDeserializer(EventType.STOP, new StopEventDataDeserializer());
         eventDeserializer.setEventDataDeserializer(EventType.GTID, new GtidEventDataDeserializer());
-        eventDeserializer.setEventDataDeserializer(EventType.WRITE_ROWS,
-                new RowDeserializers.WriteRowsDeserializer(tableMapEventByTableId));
-        eventDeserializer.setEventDataDeserializer(EventType.UPDATE_ROWS,
-                new RowDeserializers.UpdateRowsDeserializer(tableMapEventByTableId));
-        eventDeserializer.setEventDataDeserializer(EventType.DELETE_ROWS,
-                new RowDeserializers.DeleteRowsDeserializer(tableMapEventByTableId));
-        eventDeserializer.setEventDataDeserializer(EventType.EXT_WRITE_ROWS,
-                new RowDeserializers.WriteRowsDeserializer(
-                        tableMapEventByTableId).setMayContainExtraInformation(true));
-        eventDeserializer.setEventDataDeserializer(EventType.EXT_UPDATE_ROWS,
-                new RowDeserializers.UpdateRowsDeserializer(
-                        tableMapEventByTableId).setMayContainExtraInformation(true));
-        eventDeserializer.setEventDataDeserializer(EventType.EXT_DELETE_ROWS,
-                new RowDeserializers.DeleteRowsDeserializer(
-                        tableMapEventByTableId).setMayContainExtraInformation(true));
+
+        Set<Operation> skippedOperations = context.getConnectorConfig().getSkippedOps();
+        if (skippedOperations.contains("i")) {
```

Comment:
Please use `Operation.CREATE` and other constants instead of strings in these conditions.